roofit/roofit/src/RooHistConstraint.cxx

```diff
@@ -16,16 +16,14 @@
  *
  */
 
-#include "Riostream.h"
+#include <RooHistConstraint.h>
 
-#include "RooHistConstraint.h"
-#include "RooAbsCategory.h"
-#include "RooParamHistFunc.h"
-#include "RooRealVar.h"
-#include <math.h>
-#include "TMath.h"
+#include <RooParamHistFunc.h>
+#include <RooRealVar.h>
+
+#include <TMath.h>
 
-using namespace std;
+#include <math.h>
```

Comment:
I think we should include <cmath> not math.h. 
However, you can remove it, since Math.h includes cmath.

Reply:
Okay